src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.ts

```diff
@@ -144,6 +146,17 @@ export class DashboardV3Component extends PrometheusListHelper implements OnInit
   getHealth() {
     this.healthService.getMinimalHealth().subscribe((data: any) => {
       this.healthData = data;
+
+      this.nvmeofService.listGatewayGroups().subscribe({
```

Comment:
For this we need to add up and down states as well, so need to do from backend.
Similar to iscsi https://github.com/ceph/ceph/blob/d5713088a34cc1f8b8fcfd884952c24a9029ea6b/src/pybind/mgr/dashboard/controllers/health.py#L217